Modules/FT0/CMakeLists.txt

```diff
@@ -74,3 +78,17 @@ foreach(test ${TEST_SRCS})
   set_tests_properties(${test_name} PROPERTIES TIMEOUT 20)
 endforeach()
 
+# ----- Configs -------
+install(FILES
+        ft0-calibration-config.json
+        ft0-multinode-config.json
+        ft0-postprocessing-config.json
+        ft0-basic-config.json
```

Comment:
are you sure that you need these files installed ? as I understand them they are used when running manually. At P2, they will be unused as the configuration files are stored in Consul.

Reply:
Well I wanted to make more user friendly path to them, using {QUALITYCONTROL_ROOT}.

Reply:
Fair enough, please prefix them with `o2-qc-` because they will end up in the common `etc`.